redis/src/cluster_async/mod.rs

```diff
@@ -499,52 +529,209 @@ where
         Ok(slot_map)
     }
 
-    fn get_connection(&mut self, route: &Route) -> (String, ConnectionFuture<C>) {
-        if let Some((_, node_addrs)) = self.slots.range(&route.slot()..).next() {
-            let addr = node_addrs.slot_addr(route.slot_addr()).to_string();
-            if let Some(conn) = self.connections.get(&addr) {
-                return (addr, conn.clone());
-            }
+    async fn get_connection_from_parts(
```

Comment:
what from_parts means?

Reply:
where's the fix? 😅

Reply:
![image](https://media2.giphy.com/media/VbE1xtnPHx6D34GXhv/giphy.gif)